### PR TITLE
Add Mouse Atlas to clustifyrdatahub

### PR DESCRIPTION
**Pull request by SidhantPuntambekar**
This PR should add the scRNA-seq derived mouse atlas to the clustifyrdatahub repository. 

Deliverables: 

- [x] add description to R/resources.R;

- [x] add a column to inst/scripts/make-metadata.R, run that to generate inst/extdata/metadata.csv;

- [x] add inst/scripts/make-ref-data/ref_mouse_atlas.R (this doesn't have to run);

- [x] and then bump version to 0.99.4 in DESCRIPTION;

- [x] update NEWS; 

- [x] run devtools::document();

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: clustifyrdatahub
 Title: External data sets for clustifyr in ExperimentHub
-Version: 0.99.3
+Version: 0.99.4
 Description: References made from external single-cell mRNA sequencing data sets, stored as average gene expression matrices. For use with clustifyr <https://bioconductor.org/packages/clustifyr> to assign cell type identities.
 Authors@R: 
     c(person(given = "Rui",
```
Comment (raysinensis) [inline DESCRIPTION:6]: add yourself in!

File: NEWS [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Changes in version 0.99.0 (2020-07-02)
```
Comment (raysinensis) [inline NEWS:1]: Make a new line instead of changing the past change log

File: inst/scripts/make-metadata.R [hunk 3]
```diff
@@ -21,7 +22,8 @@ main.data <- data.frame(
     "Human hematopoietic cell microarray",
     "Human cortex development scRNA-seq",
     "Human pancreatic cell scRNA-seq (inDrop)",
-    "Human pancreatic cell scRNA-seq (SmartSeq2)"
+    "Human pancreatic cell scRNA-seq (SmartSeq2)",
+    "Mouse RNA-seq from 321 cell types"
```
Comment (raysinensis) [inline inst/scripts/make-metadata.R]: get the "atlas" part of the name in there

File: DESCRIPTION (hunk 1 above)
Comment (SidhantPuntambekar) [reply DESCRIPTION:6]: As an author? @raysinensis?

File: inst/scripts/make-metadata.R [hunk 4]
```diff
@@ -58,7 +61,8 @@ main.data <- data.frame(
     "TXT",
     "TSV",
     "RDA",
-    "RDA"
+    "RDA",
+    "RDS"
```
Comment (raysinensis) [inline inst/scripts/make-metadata.R]: Don't remember what they do for RDS... let's store the data as RDA. Do you already have the `ref_mouse_atlas.rda` file?

Comment (SidhantPuntambekar) [reply inst/scripts/make-metadata.R]: I think I just have it in rds form

Comment (SidhantPuntambekar) [reply inst/scripts/make-metadata.R]: Update: I now have it in RDA form: https://github.com/rnabioco/scRNA-seq-Cell-Ref-Matrix/blob/master/atlas/musMusculus/MouseAtlas.rda

File: inst/scripts/make-metadata.R (hunk 3 above)
Comment (SidhantPuntambekar) [reply inst/scripts/make-metadata.R]: See 5760d0e

File: NEWS (hunk 2 above)
Comment (SidhantPuntambekar) [reply NEWS:1]: See 5760d0e

File: inst/scripts/make-metadata.R (hunk 4 above)
Comment (SidhantPuntambekar) [reply inst/scripts/make-metadata.R]: See 93daea8

File: DESCRIPTION (hunk 1 above)
Comment (SidhantPuntambekar) [reply DESCRIPTION:6]: See 29241d2

**Review by raysinensis — APPROVED** [review]

**Pull request merged**